applications/nrf_desktop/README.rst

```diff
@@ -2067,6 +2093,12 @@ Update image verification and application image update
 Once the update image transfer is completed, the background DFU process will continue after the device reboot.
 If :ref:`nrf_desktop_config_channel_script` is used, the reboot is triggered by the script right after the image transfer completes.
 
+The implementation details of the reboot mechanism for the chosen DFU configuration are described in the subsections below.
+
+MCUboot and B0 bootloaders
```

Comment:
We have two similar headings (although on different levels) within this doc, which may cause issues. Hopefully not.